be/src/exec/schema_scanner/schema_be_compactions_scanner.h

```diff
@@ -21,7 +21,7 @@
 
 namespace starrocks {
 
-struct CompactionInfo {
+struct CompactionInfos {
```

Comment:
rename to avoid duplicated name

Reply:
I always think `info` does not have plural form like `infos`.  Do you have any idea?

Reply:
rename it to `CompactionInformation`